ui/css/itcss/tools/utilities.scss

```diff
@@ -1,37 +0,0 @@
-/*
-  Utility Classes
- */
-
-/* lib */
-.flex-column {
-  display: flex;
-  flex-direction: column;
-}
-
-.flex-row {
-  display: flex;
-  flex-direction: row;
-}
```

Comment:
`flex-row` classname is not used

![Screenshot 2025-02-07 at 2 58 18 PM](https://github.com/user-attachments/assets/0ea96d69-957d-43d3-ae5c-29ca9ac77769)